test/Splitter.t.sol

```diff
@@ -0,0 +1,433 @@
+        vow           = VowLike(ChainlogLike(LOG).getAddress("MCD_VOW"));
+        end           = EndLike(ChainlogLike(LOG).getAddress("MCD_END"));
+
+        vm.startPrank(PAUSE_PROXY);
+
+        medianizer.kiss(address(this));
+
+        stakingToken = new GemMock(1_000_000 ether);
+        farm = new StakingRewardsMock(PAUSE_PROXY, address(0), DAI, address(stakingToken));
+
+        vm.stopPrank();
+
+        splitter = Splitter(FlapperDeploy.deploySplitter({
+            deployer: address(this),
+            owner:    PAUSE_PROXY,
+            daiJoin:  DAI_JOIN,
+            farm:     address(farm)
+        }));
+
+        FlapperInstance memory flapperInstance = FlapperDeploy.deployFlapperUniV2({
+            deployer: address(this),
+            owner:    PAUSE_PROXY,
+            daiJoin:  DAI_JOIN,
+            spotter:  SPOT,
+            gem:      MKR,
+            pair:     UNIV2_DAI_MKR_PAIR,
+            receiver: PAUSE_PROXY,
+            swapOnly: true
+        });
+        flapper = FlapperUniV2SwapOnly(flapperInstance.flapper);
+
+        
+        vm.startPrank(PAUSE_PROXY);
+        // Note - this part emulates the spell initialization
+        FlapperUniV2Config memory flapperCfg = FlapperUniV2Config({
+            hop:         30 minutes,
+            want:        WAD * 97 / 100,
+            pip:         address(medianizer),
+            hump:        50_000_000 * RAD,
+            bump:        5707 * RAD,
+            pair:        UNIV2_DAI_MKR_PAIR,
+            caller:      address(splitter),
+            chainlogKey: "MCD_FLAP_BURN"
+        });
+        SplitterConfig memory splitterCfg = SplitterConfig({
+            burn:            70 * WAD / 100,
+            rewardsDuration: flapperCfg.hop,
+            farm:            address(farm),
+            chainlogKey:     "MCD_FLAP_SPLIT"
+        });
+
+        DssInstance memory dss = MCD.loadFromChainlog(LOG);
+        FlapperInit.initFlapperUniV2(dss, flapperInstance, flapperCfg);
+        FlapperInit.initSplitter(dss, address(splitter), splitterCfg);
+        FlapperInit.initDirectOracle(address(flapper));
+        vm.stopPrank();
+
+        assertEq(dss.chainlog.getAddress("MCD_FLAP_BURN"), address(flapper));
+        assertEq(dss.chainlog.getAddress("MCD_FLAP_SPLIT"), address(splitter));
+        assertEq(dss.chainlog.getAddress("FLAPPER_MOM"), address(flapperInstance.mom));
+
+        // Add initial liquidity if needed
+        (uint256 reserveDai, ) = UniswapV2Library.getReserves(UNIV2_FACTORY, DAI, MKR);
+        uint256 minimalDaiReserve = 280_000 * WAD;
+        if (reserveDai < minimalDaiReserve) {
+            changeUniV2Price(medianizer.read(), MKR, UNIV2_DAI_MKR_PAIR);
+            (reserveDai, ) = UniswapV2Library.getReserves(UNIV2_FACTORY, DAI, MKR);
+            if(reserveDai < minimalDaiReserve) {
+                topUpLiquidity(minimalDaiReserve - reserveDai, MKR, UNIV2_DAI_MKR_PAIR);
+            }
+        }
```

Comment:
Shouldn't be doing something similar to the other tests that we ensure the prices get aligned. It's built a bit different and it is true that if resetveDai < minimalDaiReverve prices will get aligned, but won't be the case with the actual pool deposits, meaning that if the oracle gets outdated the tests will start failing. I might be missing something that comes later though.

Reply:
Yes, I think it's indeed better not to rely on the oracle being up-to-date. Changed.